cloud-sql-proxy.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: cloud-sql-proxy
+  version: 2.7.0
+  epoch: 0
+  description: The Cloud SQL Auth Proxy is a utility for ensuring secure connections to your Cloud SQL instances
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - git
+      - go
+  environment:
+    CGO_ENABLED: 0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/GoogleCloudPlatform/cloud-sql-proxy
+      tag: v${{package.version}}
+      expected-commit: 2527e5aedbca617579a528070e88a6e02c4d88d1
+
+  - uses: go/build
+    with:
+      packages: .
+      output: cloud-sql-proxy
+      ldflags: -X github.com/GoogleCloudPlatform/cloud-sql-proxy/v2/cmd.metadataString=container
+
+  - uses: strip
```

Comment:
Can you add a `compat` subpackage that depends on this, and makes the binary available with the expected name in `/`?

We should then add the compat package as a runtime dependency of https://github.com/wolfi-dev/os/blob/8f59ae1ede83ca7fce746b9610b09594b026ff9e/sigstore-scaffolding.yaml#L27

Reply:
is the expected name `cloud-sql-proxy` or `cloudsqlproxy`?

is this what you had in mind?